site/content/docs/include/common-svc-fields.en.md

```diff
@@ -34,7 +34,7 @@ Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs
 <div class="separator"></div>
 
 <a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
-Non-default operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`.
+Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86_64`.
```

Comment:
Looks like this file isn't used any where anymore. We should probably just remove it some day after we make sure it's not referenced at all.